internal/network/service.go

```diff
@@ -56,6 +56,10 @@ func ReconcileInputService(er record.EventRecorder, k8sClient client.Client, nam
 		visitors,
 	)
 
+	if !isDaemonset {
+		desired.Spec.Selector["collector-resource-type"] = "deployment"
```

Comment:
maybe... 'collector-deployment-kind' ... and didn't I see a constant for the value